<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuehleGUI</class>
 <widget class="QMainWindow" name="MuehleGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MuehleGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PB_start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_verbinden">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Verbinden</string>
    </property>
   </widget>
   <widget class="QLabel" name="hostLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>55</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Host:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>55</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_host">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>461</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_port">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>460</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_isServer">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>465</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_NNQ1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>NNQ1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_NNQ2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>NNQ2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_NNQ3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>NNQ3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_NOQ1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>NOQ1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_NOQ2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>NOQ2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_NOQ3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>NOQ3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_OOQ1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>OOQ1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_OOQ2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>OOQ2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_OOQ3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>OOQ3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_SOQ1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SOQ1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_SOQ2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SOQ2</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_SOQ3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SOQ3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_SWQ3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SWQ3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_WWQ3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>WWQ3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_WWQ2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>WWQ2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_WWQ1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>WWQ1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_NWQ3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>NWQ3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_NWQ2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>NWQ2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_NWQ1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>NWQ1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_SSQ3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SSQ3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_SSQ2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SSQ2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_SSQ1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SSQ1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_SWQ2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SWQ2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_SWQ1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SWQ1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
